### PR TITLE
feat: support params

### PR DESCRIPTION
**Pull request by Garbee**
The absolute most minimal thing we can do to support generic `params`. Nothing to do with updating should any attributes change.

Also takes advantage of `URLSearchParams` as discussed in #28. This makes the work we do far cleaner and less error prone. Really nice that the constructor takes a string to start the data off with.

### CODE REVIEW

**Comment by achinchen** [comment]
It's what i'm looking for!
Just wonder when this PR will be merged 😅

File: src/lite-yt-embed.js [hunk 1]
```diff
@@ -99,10 +99,12 @@ class LiteYTEmbed extends HTMLElement {
     }
 
     addIframe(){
+        const params = new URLSearchParams(this.getAttribute('params'));
```
Comment (paulirish) [inline src/lite-yt-embed.js:102]:
```suggestion
        const params = new URLSearchParams(this.getAttribute('params') || []);
```

if nothing was defined, we'd be using the `null` return value.. 

image

File: variants/params.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="../src/lite-yt-embed.css" />
+</head>
+<body>
+<h1><code>lite-youtube</code> custom element</h1>
+
+<p>
+    This is embedded with custom parameters to make the playback controls not display on the video.
+    You may use any parameters <a href="https://developers.google.com/youtube/player_parameters">supported by YouTube</a> that are applicable to a video embed.
```
Comment (paulirish) [inline variants/params.html:14]:
```suggestion
    You may use any <a href="https://developers.google.com/youtube/player_parameters#Parameters">parameters supported by the YouTube IFrame Player API</a>.
```

**Review by paulirish — APPROVED** [review]
this is super clean.  i love me some `URLSearchParams`

**Pull request merged**